### PR TITLE
Integrate VAnchor Deposit

### PR DESCRIPTION
**Pull request by nepoche**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [ ] Tests for the changes have been added (for bug fixes/features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [x] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Other... Please describe:

## What is the current behavior?

No VAnchor exists

Issue Number: N/A

## What is the new behavior?
Add support for VAnchor deposit integration

## Does this PR introduce a breaking change?
- [ ] Yes
- [ ] No

### CODE REVIEW

File: packages/apps/src/configs/chains/chain-config.ts [hunk 1]
```diff
@@ -88,34 +88,34 @@ export const chainsConfig: AppConfig['chains'] = {
...
+    name: 'Arbitrum Testnet',
+    url: 'https://rinkeby.arbitrum.io/rpc',
+    evmRpcUrls: ['https://rinkeby.arbitrum.io/rpc'],
+    blockExplorerStub: 'https://testnet.arbiscan.io',
+    logo: ArbitrumLogo,
+    tag: 'test',
+    currencies: [WebbCurrencyId.ETH, WebbCurrencyId.WETH, WebbCurrencyId.webbWETH],
+    nativeCurrencyId: WebbCurrencyId.ETH,
+  },
   // [InternalChainId.HarmonyTestnet1]: {
```
Comment (dutterbutter) [inline packages/apps/src/configs/chains/chain-config.ts:119]: Can we remove the commented out code?

File: packages/bridge/src/components/DepositConfirm/DepositConfirm.tsx [hunk 2]
```diff
@@ -210,6 +210,7 @@ export const DepositConfirm: React.FC<DepositInfoProps> = ({
             if (!depositPayload) {
               return;
             }
+            console.log('depositPayload: ', depositPayload);
```
Comment (dutterbutter) [inline packages/bridge/src/components/DepositConfirm/DepositConfirm.tsx:213]: Please remove the console log

File: tsconfig.json [hunk 3]
```diff
@@ -32,107 +32,113 @@
 	"skipDefaultLibCheck": true,
 	"paths": {
 	  "@webb-dapp/apps/*": [
-		"apps/src/*"
+			"apps/src/*"
```
Comment (dutterbutter) [inline tsconfig.json:35]: Might want to fix formatting?

File: packages/vbridge/src/components/DepositConfirm/DepositConfirm.tsx [hunk 4]
```diff
@@ -0,0 +1,228 @@
...
+        />
+
+        <SpaceBox height={8} />
+
+        <MixerButton
+          onClick={async () => {
+            if (!depositPayload) {
+              return;
+            }
+            console.log('depositPayload: ', depositPayload);
```
Comment (dutterbutter) [inline packages/vbridge/src/components/DepositConfirm/DepositConfirm.tsx:215]: Can we remove the console log

File: tsconfig.json (hunk 3 above)
Comment (nepoche) [reply tsconfig.json:35]: I'm not sure why the git diff is making it look worse - when I check the source code in my editor as well as the code at the particular commit, these changes are actually fixing formatting.

File: packages/vbridge/src/components/DepositConfirm/DepositConfirm.tsx (hunk 4 above)
Comment (nepoche) [reply packages/vbridge/src/components/DepositConfirm/DepositConfirm.tsx:215]: I actually like the console log here on the deposit payload - it shouldn't be spammy and is informative.

**Pull request merged**